frontend/packages/operator-lifecycle-manager/src/components/descriptors/spec/index.tsx

```diff
@@ -13,6 +13,7 @@ import { ResourceRequirementsModalLink } from './resource-requirements';
 import { EndpointList } from './endpoint';
 import { configureSizeModal } from './configure-size';
 import { configureUpdateStrategyModal } from './configure-update-strategy';
+import { REGEXP_K8S_RESOURCE_SUFFIX } from '../../operand/const';
```

Comment:
@TheRealJon I suspect this is pulling in react-jsonschema-form to the operatorhub bundle

Reply:
Hmm, okay let me see if I can figure something out.

Reply:
Might move those regex constants into the descriptor folder since they relate more closely to descriptors than operands.